### ISSUE TITLE
error trying to install from 64 bit windows command line

### ISSUE THREAD

**Issue by wcb98** [renpy/pygame_sdl2#107]
C:/...>py -2 setup.py install

Traceback (most recent call last):
  File "setup.py", line 60, in <module>
    parse_cflags([ "sh", "-c", "sdl2-config --cflags" ])
  File "C:\Users\William\Downloads\pygame_sdl2-master\setuplib.py", line 93, in parse_cflags
    output = subprocess.check_output(command, universal_newlines=True)
  File "C:\Python27\lib\subprocess.py", line 216, in check_output
    process = Popen(stdout=PIPE, *popenargs, **kwargs)
  File "C:\Python27\lib\subprocess.py", line 394, in __init__
    errread, errwrite)
  File "C:\Python27\lib\subprocess.py", line 644, in _execute_child
    startupinfo)
WindowsError: [Error 2] The system cannot find the file specified

Any ideas what's wrong?

**Comment by Meithal** [comment]
You must install the windeps first
https://github.com/renpy/pygame_sdl2#windows

**Comment by GeraShrek** [comment]
> You must install the windeps first
> https://github.com/renpy/pygame_sdl2#windows

Okay, I downloaded that. What do next? There is the same mistake WindowsError: [Error 2] The system cannot find the file specified

**Comment by LeehamElectronics** [comment]
Same here...

**Comment by keenfoong** [comment]
Also getting this error, what else would I need to do here?

```
pygame_sdl2.error is out of date.
Traceback (most recent call last):
  File "D:\Code\pygame_sdl2\pygame_sdl2\setup.py", line 110, in <module>
    cython("pygame_sdl2.error", libs=sdl_libs)
  File "D:\Code\pygame_sdl2\pygame_sdl2\setuplib.py", line 246, in cython
    subprocess.check_call([
  File "C:\Users\keenf\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 368, in check_call
    retcode = call(*popenargs, **kwargs)
  File "C:\Users\keenf\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Users\keenf\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\keenf\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

**Comment by tanyeun** [comment]
It seems that the DEV are not so much care about windows developers.

I ran into the same issue. Failed to comple pygame_sdl2_windeps installation

python makelib.py lib/x86/libpng16-16.dll lib/x86/png.lib x86
python makelib.py lib/x86/libjpeg-9.dll lib/x86/jpeg.lib x86
python makelib.py lib/x64/libpng16-16.dll lib/x64/png.lib x64
python makelib.py lib/x64/libjpeg-9.dll lib/x64/jpeg.lib x64

these commands all failed for the same reason, file not found.
I tried to find secure source to download libp*.dll but failed.
DLL is notoriously to hide viruses so I am hesitated to download from non-trusted source.
